examples-watchdog/CMakeLists.txt

```
@@ -3,9 +3,9 @@ add_executable(patch_eeprom patch_eeprom.c)
 target_link_libraries(patch_eeprom ftdi1)
 
 install(TARGETS patch_eeprom
-  DESTINATION /home/pi/connect/
+  DESTINATION /usr/lib/revpi-tools/
```

Suggestion: Use `CMAKE_INSTALL_*` variables:

```suggestion
  DESTINATION ${CMAKE_INSTALL_LIBDIR}/revpi-tools
```

https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html#result-variables